[ 2025-03-10 10:36:54,665 ] 59 NetworkSecurity - ERROR - Uncaught exception
Traceback (most recent call last):
  File "/home2/cslab/Videos/MlOps-Network-Security-Analysis/main.py", line 1, in <module>
    from networksecurity.components.data_ingestion import DataIngestion
ImportError: cannot import name 'DataIngestion' from 'networksecurity.components.data_ingestion' (/home2/cslab/Videos/MlOps-Network-Security-Analysis/networksecurity/components/data_ingestion.py)
